data/projects.js

```diff
@@ -81,5 +81,51 @@ coala should support generating metrics for your code.",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Core", "Bears"],
 		"markdown": "implement_metrics_for_coala.md"
+	},
+	{
+		"name" : "Debug and Profile Bears",
```

Comment:
I like this although I'm a bit sceptical about "profiling" as in "performance".
The overhead to running a linter that is currently introduced by coala is immense.